### ISSUE TITLE
Ctrl+F not working. Find from menu does, though.

### ISSUE THREAD

**Issue by hossameldeen** [beakerbrowser/beaker#1219]
Operation System: Ubuntu 16.04
Beaker version: Source-latest (7bac87ca9de26289e042b4bd382027dac6ceab4e)

## Problem

Ctrl+F not working. "Find in page" from menu works, though.

## Relevant

- This happened after #1189's fix.
- This also happened when [we tried to upgrade electron-localshortcut to 3.x](https://github.com/beakerbrowser/beaker/issues/1189#issuecomment-428025833).
- The shortcuts in the UI menu read stuff like "^F". Not sure if this has always been the case; just mentioning it in case it helps.
![image](https://user-images.githubusercontent.com/6719916/47251673-e0d24800-d436-11e8-88b2-145320260034.png)

Opened it as a separate issue from #1189 as it's a different problem.

And really, it's no big deal for me and not urgent at all :). And thanks again for fixing #1189, that was awesome! and it eliminated a pain point indeed.

**Comment by pfrazee** [comment]
Thanks for reporting, I'll check into it.

**Comment by hossameldeen** [comment]
I'm not intending to spend time debugging it for now, it's not that much of an issue for me currently. When I have time, will see how to extract useful Ubuntu logs to debug this problem (event listeners on Ubuntu level, .. etc).

For now, to anyone reading this:

1) If you're experiencing the problem as well, please tell us to make sure it's not a fault at my side.
2) If you know how to get useful Ubuntu logs to debug this issue, please tell us.

**Comment by pfrazee** [comment]
@hossameldeen are there any other shortcuts that dont work, by chance?

**Comment by puf17640** [comment]
@pfrazee Does beaker internally log out errors? If so, where could one find them?

**Comment by pfrazee** [comment]
`beaker://debug-log/`

**Comment by puf17640** [comment]
Thanks! Since you didn't mention it, I guess, you won't find any errors related to this issue in the log? That was my first thought, but ubuntu's error logs might be more helpful.

**Comment by pfrazee** [comment]
Probably not. The debug-log is *really* noisy right now and I might remove some of the dat events because we now have the swarm debugger.

**Comment by hossameldeen** [comment]
@pfrazee You're right, some other shortcuts don't work. For the menu shortcuts:

- Ctrl+N, T work.
- Ctrl+F, H, O don't.

I don't know what other shortcuts there are, but Ctrl+number and Ctrl+W work.

**Comment by pfrazee** [comment]
The ones that work seem to be the ones registered using our `localshortcut` module. The ones that fail are the keyboard accelerators. I'd bet a nickle it's related to https://github.com/electron/electron/pull/15094.

**Comment by hossameldeen** [comment]
Your work on `localshortcut` wasn't in vain indeed, @pfrazee :-)

**Comment by gabefair** [comment]
I am also encountering this issue. My logs are below:
>beaker Uncaught exception: Error: Must provide a non-empty search content
    at WebContents.findInPage (C:\Users\gabri\AppData\Local\Programs\beaker-browser\resources\electron.asar\browser\api\web-contents.js:321:15)
    at View.moveInpageFind (C:\Users\gabri\AppData\Local\Programs\beaker-browser\resources\app.asar\background-process.build.js:2494:22)
    at View.showInpageFind (C:\Users\gabri\AppData\Local\Programs\beaker-browser\resources\app.asar\background-process.build.js:2471:12)
    at click (C:\Users\gabri\AppData\Local\Programs\beaker-browser\resources\app.asar\background-process.build.js:1847:26)
    at MenuItem.click (C:\Users\gabri\AppData\Local\Programs\beaker-browser\resources\electron.asar\browser\api\menu-item.js:56:9)
    at Function.executeCommand (C:\Users\gabri\AppData\Local\Programs\beaker-browser\resources\electron.asar\browser\api\menu.js:31:13)
